daemon/src/lib.rs

```diff
@@ -476,6 +477,10 @@ where
             .context("Price feed not available")?
             .context("No quote available")?;
 
+        if latest_quote.is_older_than(1.minutes()) {
```

Comment:
🤔 Perhaps a constant would make sense here. Also the error message could change depending on its value.